src/main/java/org/sonarlint/intellij/ui/nodes/FindingNode.kt

```diff
@@ -21,4 +21,9 @@ package org.sonarlint.intellij.ui.nodes
 
 import org.sonarlint.intellij.finding.Finding
 
-abstract class FindingNode(val finding: Finding): AbstractNode()
+abstract class FindingNode(val finding: Finding): AbstractNode() {
+
+    override fun getFindingCount(): Int {
+        return 1
+    }
```

Comment:
Another way of writing this:
```suggestion
override fun getFindingCount() = 1
```